cmd/config/internal/commands/cmdset.go

```diff
@@ -32,18 +34,48 @@ func NewSetRunner(parent string) *SetRunner {
 		"annotate the field with who set it")
 	c.Flags().StringVar(&r.Perform.Description, "description", "",
 		"annotate the field with a description of its value")
+	c.Flags().StringVar(&setterVersion, "version", "",
+		"use this version of the setter format")
+	c.Flags().MarkHidden("version")
 
 	return r
 }
 
+var setterVersion string
+
+var GetOpenAPIFile = func(args []string) (string, error) {
+	return filepath.Join(args[0], "kustomization"), nil
```

Comment:
Should this be kustomization.yaml.

Reply:
no, the .yaml suffix creates issues.  should follow up on this.